pagopa/io-app
# fix: [PE-801] CGN WebView crash error

## Short description
This PR fixes a critical bug in the CGN WebView for certain merchants. A `404` error following a recent library update caused the entire event object to be sent to Mixpanel as a property. This led to app crashes, as the event object contained not only read-only data but also methods that were not correctly serialized when sending the event to Mixpanel. Additionally, the old legacy `GenericErrorComponent` has been replaced with the new `OperationResultScreen` component.

## List of changes proposed in this pull request
- Removed the spread operator to send the entire `event` property to mixpanel when an error occurs, just the description if available;
- Replaced the legacy `GenericErrorComponent` with the `OperationResultScreenContent`;

## How to test
- With the app builded in production env, open the CGN operator "Centro Linguistico Moose" and then select the discount "Test di ingresso, test finale e attestato" and check that the app doesn't crash.

**N.B:** The error occurs because the merchant provided an old url that isn't available anymore that has a 404 response.

## Preview
|Before|After|
|-|-|
|<video src="https://github.com/user-attachments/assets/5740fe67-8acb-4549-9593-c37edaed9cc1" />|<video src="https://github.com/user-attachments/assets/8ff83499-9c3b-4ed4-bb10-a04e19a36d33" />
